### PR TITLE
[zh-cn]: create the translation of `VirtualKeyboard.overlaysContent` property

### PR DESCRIPTION
**Pull request by tianyeeT**
…property

### CODE REVIEW

File: files/zh-cn/web/api/virtualkeyboard/overlayscontent/index.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+slug: Web/API/VirtualKeyboard/overlaysContent
+l10n:
+  sourceCommit: 5cdb341c723de0edb273769555d9124266d9c851
+---
+
+{{APIRef("VirtualKeyboard API")}}{{SeeCompatTable}}{{securecontext_header}}
+
+VirtualKeyboard 接口的 **`overlaysContent`** 属性用于选择退出浏览器自动处理屏幕虚拟键盘的方式，通过减小视窗大小为其腾出空间。
+
+如果将 `overlaysContent` 属性设置为 `true`，则浏览器在虚拟键盘出现时不再调整视窗大小。而是让虚拟键盘覆盖在网页内容上，你可以使用 {{domxref("VirtualKeyboard_API", "Virtual Keyboard API", "", "nocode")}}、自定义的 CSS 和 JavaScript 适应页面布局。
```
Comment (PassionPenguin) [inline files/zh-cn/web/api/virtualkeyboard/overlayscontent/index.md:12]:
```suggestion
如果将 `overlaysContent` 属性设置为 `true`，则浏览器在虚拟键盘出现时不再调整视窗大小，而是让虚拟键盘覆盖在网页内容上。从而，你可以使用 {{domxref("VirtualKeyboard_API", "Virtual Keyboard API", "", "nocode")}}、自定义的 CSS 和 JavaScript 适应页面布局。
```

Comment (skyclouds2001) [inline files/zh-cn/web/api/virtualkeyboard/overlayscontent/index.md:12]: 参见 https://github.com/mdn/translated-content/pull/22039/files#r1660418839

File: files/zh-cn/web/api/virtualkeyboard/overlayscontent/index.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+slug: Web/API/VirtualKeyboard/overlaysContent
+l10n:
+  sourceCommit: 5cdb341c723de0edb273769555d9124266d9c851
+---
+
+{{APIRef("VirtualKeyboard API")}}{{SeeCompatTable}}{{securecontext_header}}
+
+VirtualKeyboard 接口的 **`overlaysContent`** 属性用于选择退出浏览器自动处理屏幕虚拟键盘的方式，通过减小视窗大小为其腾出空间。
+
+如果将 `overlaysContent` 属性设置为 `true`，则浏览器在虚拟键盘出现时不再调整视窗大小，而是让虚拟键盘覆盖在网页内容上。从而，你可以使用 {{domxref("VirtualKeyboard_API", "Virtual Keyboard API", "", "nocode")}}、自定义的 CSS 和 JavaScript 适应页面布局。
```
Comment (tianyeeT) [inline files/zh-cn/web/api/virtualkeyboard/overlayscontent/index.md:12]:
```suggestion
如果将 `overlaysContent` 属性设置为 `true`，则浏览器在虚拟键盘出现时不再调整视口大小，而是让虚拟键盘覆盖在网页内容上。从而，你可以使用 {{domxref("VirtualKeyboard_API", "Virtual Keyboard API", "", "nocode")}}、自定义的 CSS 和 JavaScript 适应页面布局。
```

File: files/zh-cn/web/api/virtualkeyboard/overlayscontent/index.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+---
+title: VirtualKeyboard：overlaysContent 属性
+slug: Web/API/VirtualKeyboard/overlaysContent
+l10n:
+  sourceCommit: 5cdb341c723de0edb273769555d9124266d9c851
+---
+
+{{APIRef("VirtualKeyboard API")}}{{SeeCompatTable}}{{securecontext_header}}
+
+VirtualKeyboard 接口的 **`overlaysContent`** 属性用于选择退出浏览器自动处理屏幕虚拟键盘的方式，通过减小视窗大小为其腾出空间。
```
Comment (yin1999) [inline files/zh-cn/web/api/virtualkeyboard/overlayscontent/index.md:10]:
```suggestion
VirtualKeyboard 接口的 **`overlaysContent`** 属性用于选择不使用浏览器自动处理屏幕虚拟键盘的方式（减小视窗大小来为屏幕虚拟键盘腾出空间）。
```

File: files/zh-cn/web/api/virtualkeyboard/overlayscontent/index.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+如果将 `overlaysContent` 属性设置为 `true`，则浏览器在虚拟键盘出现时不再调整视口大小，而是让虚拟键盘覆盖在网页内容上。从而，你可以使用 {{domxref("VirtualKeyboard_API", "Virtual Keyboard API", "", "nocode")}}、自定义的 CSS 和 JavaScript 适应页面布局。
```
Comment (yin1999) [inline files/zh-cn/web/api/virtualkeyboard/overlayscontent/index.md:12]:
```suggestion
如果将 `overlaysContent` 属性设置为 `true`，则浏览器在虚拟键盘出现时不再调整视口大小，而是让虚拟键盘覆盖在网页内容上。从而，你可以使用{{domxref("VirtualKeyboard_API", "虚拟键盘 API", "", "nocode")}}、自定义 CSS 和 JavaScript 适应页面布局。
```

File: files/zh-cn/web/api/virtualkeyboard/overlayscontent/index.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+## 值
+
+一个布尔值。默认为 `false`，设置为 `true` 可选择退出浏览器对屏幕虚拟键盘的自动处理。
```
Comment (yin1999) [inline files/zh-cn/web/api/virtualkeyboard/overlayscontent/index.md:16]:
```suggestion
一个布尔值。默认为 `false`，设置为 `true` 可选择不使用浏览器对屏幕虚拟键盘的自动处理。
```

File: files/zh-cn/web/api/virtualkeyboard/overlayscontent/index.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+## 示例
+
+```js
+if ("virtualKeyboard" in navigator) {
+  console.log(navigator.virtualKeyboard.overlaysContent); // false
+  navigator.virtualKeyboard.overlaysContent = true; // 选择退出自动处理。
```
Comment (yin1999) [inline files/zh-cn/web/api/virtualkeyboard/overlayscontent/index.md:23]:
```suggestion
  navigator.virtualKeyboard.overlaysContent = true; // 选择不使用自动处理。
```

**Review by yin1999 — COMMENTED** [review]
一些建议

**Review by yin1999 — APPROVED** [review]

**Pull request merged**